### DISCUSSION TITLE
Montage with different image sizes, how to resize so all zoom to fit without white space?

### DISCUSSION THREAD

**Discussion by AWO52PZ44IKE7FDR79DV2999BCZK19R0IFLDGHF** [ImageMagick/ImageMagick#2024] [Help] [answered]
Hi there, I am having a bit of an issue here. I have 6 images all different sizes. There is one image that just wont fit correctly. Here is an example.

Here are my images: https://imgur.com/a/0N9KB6B

The best I can do is this: 
`magick montage -adjoin -geometry 500x5000x1 -mode Concatenate -tile 3x3 * preview.png`
![image](https://i.imgur.com/5Ojpqos.png) 

I need the bottom image to zoom to fit. This is what I am looking for:
![image](https://i.imgur.com/gPQN0lu.png) 

Is this possible? Any help is greatly appreciated!

**Comment by snibgo** [comment, marked as answer]
What version of IM, on what platform?

You use single-colour images so I'm not sure what you want, but how about:

`magick montage -adjoin -resize "300x500^^" -gravity Center -crop 300x500+0+0 -mode Concatenate -tile 3x3 *.jpg out.png`

This is Windows syntax, where "^" is a special character, so needs to be escaped with another "^".

**Reply by AWO52PZ44IKE7FDR79DV2999BCZK19R0IFLDGHF** [reply]
Windows, and that's great, I was actually trying this method earlier and I couldn't figure out how to escape properly for windows. You have helped a ton. Thank you
